src/launch_application.c

```diff
@@ -1075,15 +1075,15 @@ static void SetName(TaskManager *taskman) {
     FieldSystem *fieldSystem = TaskManager_GetFieldSystem(taskman);
     NamingScreenData *data   = TaskManager_GetEnvironment(taskman);
     switch (data->args->unk0) {
-    case 0:
+    case NAME_SCREEN_PLAYER:
         PlayerProfile *profile = Save_PlayerData_GetProfileAddr(fieldSystem->saveData);
         Save_Profile_PlayerName_Set(profile, data->args->unk1C);
```

Comment:
unk1C -> name (playerNameInputString?)

Reply:
nvm it also gets used for pokemon names, should theoretically also be nameInputString but that's already used for the rival so ???

Reply:
out of scope